node/core/approval-voting/src/import.rs

```diff
@@ -882,13 +918,30 @@ pub(crate) mod tests {
 			.collect::<Vec<_>>();
 
 		let test_fut = {
-			let session_window =
-				RollingSessionWindow::with_session_info(session, vec![session_info]);
+			let session_info_provider = SessionInfoProvider {
+				runtime_info: RuntimeInfo::new_with_cache(
+					RuntimeInfoConfig {
+						keystore: None,
+						session_cache_lru_size: NonZeroUsize::new(DISPUTE_WINDOW.get() as usize)
```

Comment:
`SessionWindowSize` should have a conversion function to `NonZeroUsize` - or alternatively even better be an `NonZeroUsize` newtype wrapper - if that's possible.

Reply:
I've added a conversion function. I've tried wrapping `NonZeroUsize` instead of `SessionIndex` but I hit a wall with
https://github.com/paritytech/polkadot/blob/233478e1987886c58e8f9030a6fa221d43035f82/node/primitives/src/lib.rs#L141
For `NonZeroUsize` it's an unsafe function so this won't work without an `expect()`. Maybe we can do something smarter but better handle it separately.

Should I open an issue for this?

Reply:
If you hit a wall it is probably not worth pursuing right now. Thanks for checking!